modules/mobile/app/services/mobile/v0/claims/proxy.rb

```diff
@@ -1,7 +1,5 @@
 # frozen_string_literal: true
 
-require 'lighthouse/facilities/client'
```

Comment:
I believe we refactored this a few months ago and this is no longer needed in this file.